### ISSUE TITLE
[bug] The `chainImages` configuration is not working.

### ISSUE THREAD

**Issue by baijunjie** [reown-com/appkit#3057]
### Link to minimal reproducible example

https://github.com/baijunjie/reown-appkit-issue-demo

### Summary

https://docs.reown.com/appkit/vue/core/options#chainimages

```js
createAppKit({
  ...
  chainImages: {
    "31337": "http://localhost:3000/img/chainIcons/hardhat.svg",
    "42161": "http://localhost:3000/img/chainIcons/arbitrum.svg",
  },
  ...
})
```

<img width="331" alt="image" src="https://github.com/user-attachments/assets/d26b470c-b664-4c98-9b13-96c6fc72d166">

Expected icon:
<img width="209" alt="image" src="https://github.com/user-attachments/assets/f52085b9-3474-4ee4-910f-9caf7fcd741e">

### List of related npm package versions

```
"@reown/appkit": "^1.1.0",
"@reown/appkit-adapter-wagmi": "^1.1.0",
```

**Comment by rtomas** [comment]
https://linear.app/reown/issue/APKT-1292/check-functionality-of-chainimages-feature

**Comment by rtomas** [comment]
Fixed in v1.1.8

Thanks for the patience
